INSTALL

```diff
@@ -37,5 +37,4 @@ a list of all the operational servers in your account:
     - ~/.ssh/id_rsa
     :user: <email>
     :refresh_token: <refresh_token>
-    :api_url: https://us-3.rightscale.com
-    :account: <account>
+    :api_url: https://us-3.rightscale.com/api/acct/<account>
```

Comment:
Let's leave the `:account` part in as I think some of the other tools might use it.

Reply:
Correct - this would possibly break the normal 'user/password' login method.

Reply:
Correct - this would possibly break the normal 'user/password' login method.

Reply:
Disagree.  This is specifically for the "RIGHTSCALE OAUTH SPECIFIC NOTES" (aka refresh_token) part of the instructions, not related to username/pass (which if you look at the file, you'll see doesn't have a ":account:' field)

Reply:
Have you checked that all of the release automation tools that use this file actually always just use `:api_url` only and never use `:account`? When I set up these files for using release automation the account number was in both places, that may be cargo cult behavior, but it may also actually be required.

Reply:
I don't understand the pushback.  As I noted in the PR request, this does not work for me when using refresh_token:

:api_url: https://us-3.rightscale.com
:account: 9202

This does:

:api_url: https://us-3.rightscale.com/ap/acct/9202

The error from right_api_one_six was included in the PR

Reply:
Because:
```yaml
:api_url: https://us-3.rightscale.com/ap/acct/9202
:account: 9202
```
will also work, and will probably not break any other tools that may still be relying on the `:account` value being in the config.

Reply:
Updated